modules/brittbot/bang.py

```diff
@@ -28,6 +28,14 @@ def pod_bay_doors(jenni, input):
 pod_bay_doors.rule = "^!?open the pod bay doors(, $nickname)?"
 pod_bay_doors.priority = 'medium'
 
+@smart_ignore
+def party(jenni, input):
+    jenni.say("/me dances :D-<")
+    jenni.say("/me dances :D|-<")
+    jenni.say("/me dances :D/-<")
+    jenni.say("/me dances :D\-<")
```

Comment:
Lets keep each function limited to one say call to prevent spam and the bot from being kicked by irc services.